.github/workflows/qa.yaml

```diff
@@ -163,7 +163,7 @@ jobs:
       - name: Install VHS and ttyd for integration tests
         run: |
           set -eu
-          go install github.com/charmbracelet/vhs@latest
+          go install github.com/charmbracelet/vhs@2b75966
```

Comment:
Please mention in the commit message why we're pinning a specific commit here

Reply:
Leaving this open so that I will do at final rebase time.